packages/devtools-source-map/package.json

```diff
@@ -8,12 +8,19 @@
   "bugs": "https://github.com/devtools-html/devtools-core/issues",
   "homepage": "https://github.com/devtools-html/devtools-core/tree/master/packages/devtools-source-map#readme",
   "main": "src/index.js",
+  "scripts": {
+    "copy-assets": "node bin/copy-assets"
+  },
   "dependencies": {
     "devtools-client-adapters": "^0.0.21",
     "devtools-config": "^0.0.12",
     "devtools-network-request": "^0.0.14",
     "md5": "^2.2.1",
     "source-map": "^0.5.6",
     "url": "^0.11.0"
+  },
+  "devDependencies": {
+    "devtools-launchpad": "^0.0.56",
+    "webpack": "1.14.0"
```

Comment:
Ah, I had thought `copy-assets` was explicitly requiring `webpack` itself, but looks like it is indeed happening via `devtools-launchpad` instead. Ok, I'll remove it here.